lib/shelley/src/Cardano/Wallet/Shelley.hs

```diff
@@ -386,16 +359,15 @@ serveWallet
         -> NetworkLayer IO (CardanoBlock StandardCrypto)
         -> (WorkerCtx (ApiLayer s k) -> WalletId -> IO ())
         -> IO (ApiLayer s k)
-    apiLayer tl nl coworker = do
-        let params = (block0, np, sTolerance)
+    apiLayer txLayer netLayer coworker = do
         tokenMetaClient <- newMetadataClient tokenMetadataTracer tokenMetaUri
-        db <- Sqlite.newDBFactory
+        dbFactory <- Sqlite.newDBFactory
             walletDbTracer
             (DefaultFieldValues
                 { defaultActiveSlotCoefficient =
-                    getActiveSlotCoefficient sp
+                    getActiveSlotCoefficient slottingParameters
```

Comment:
I think that the use of abbreviations like `sp` is mainly due to Hlint complaining about name shadowing.